packages/sync-core/src/lib/TLSocketRoom.ts

```diff
@@ -268,6 +268,14 @@ export class TLSocketRoom<R extends UnknownRecord = UnknownRecord, SessionMeta =
 		return this.room.getSnapshot()
 	}
 
+	/**
+	 * Return a serialized snapshot of the document state, including clock-related bookkeeping.
+	 * @returns The serialized snapshot
+	 */
+	getCurrentSerializedSnapshot() {
+		return JSON.stringify(this.room.getSnapshot())
+	}
```

Comment:
let's mark this as internal for now. don't want to pollute the public api with trivial helper stuff